src/storage/txn/commands/acquire_pessimistic_lock.rs

```diff
@@ -60,14 +60,17 @@ impl CommandExt for AcquirePessimisticLock {
     gen_lock!(keys: multiple(|x| &x.0));
 }
 
-fn extract_lock_from_result<T>(res: &StorageResult<T>) -> Lock {
+fn extract_lock_key_from_result<T>(res: &StorageResult<T>) -> (Lock, Vec<u8>) {
```

Comment:
Can we just return `info` for this fuction, and let the only user(` process_write`) do the rest part?